src/zenml/zen_stores/models/stack_wrapper.py

```diff
@@ -51,3 +52,13 @@ def to_stack(self) -> Stack:
         return Stack.from_components(
             name=self.name, components=stack_components
         )
+
+    def get_component(
```

Comment:
What do you think about calling this method `get_component_wrapper`? Sometimes it gets quite challenging to read the code as there are already multiple naming conventions. This might increase the clarity a bit.

Reply:
Done